sdk-setup/src/sdk-manage

```diff
@@ -489,12 +489,13 @@ get_all_or_selected_of_type() {
         done
     fi
 
-    # Exclude snapshots
+    # Exclude snapshots unless required explicitly
     if [[ $type == target ]]; then
         local objects_=("${objects[@]}")
         objects=()
         for object in "${objects_[@]}"; do
-            target_is_snapshot "$object" || objects+=("$object")
+            ! target_is_snapshot "$object" || ! [[ $get_all ]] || continue
```

Comment:
This makes my brain hurt. But I *think* the logic is correct.

Reply:
Whenever you see `!A || B`, read it as `A -> B` :)

Reply:
Yes, I know. Still having two of those in a row is too much for me, at least on a friday afternoon :)